slack_sdk/web/client.py

```diff
@@ -4686,6 +4686,34 @@ def team_billing_info(
         """
         return self.api_call("team.billing.info", params=kwargs)
 
+    def team_externalTeams_list(
+        self,
+        *,
+        connection_status_filter: Optional[str] = None,
+        slack_connect_pref_filter: Optional[List[str]] = None,
+        sort_direction: Optional[str] = None,
+        sort_field: Optional[str] = None,
+        workspace_filter: Optional[List[str]] = None,
+        cursor: Optional[str] = None,
+        limit: Optional[int] = None,
+        **kwargs,
+    ) -> SlackResponse:
+        """Gets all the external teams connected with their connection detail.
```

Comment:
The latest description seems to be:
```suggestion
        """Returns a list of all the external teams connected and details about the connection.
```